src/Azure.Functions.Cli/Helpers/ZipHelper.cs

```diff
@@ -48,17 +48,22 @@ public static Stream CreateZip(IEnumerable<string> files, string rootPath)
         {
             const int defaultBufferSize = 4096;
             var fileStream = new FileStream(Path.GetTempFileName(), FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite, defaultBufferSize, FileOptions.DeleteOnClose);
-
-            using (var zip = new ZipArchive(fileStream, ZipArchiveMode.Create, leaveOpen: true))
+            using (ZipFile zip = new ZipFile())
             {
+                zip.CompressionLevel = Ionic.Zlib.CompressionLevel.BestSpeed;
                 foreach (var fileName in files)
                 {
                     zip.AddFile(fileName, fileName, rootPath);
                 }
+                zip.Save(fileStream);
             }
-
             fileStream.Seek(0, SeekOrigin.Begin);
             return fileStream;
         }
+
+        public static string FixFileNameForZip(this string value, string zipRoot)
```

Comment:
no longer needed, right?

Reply:
Oops! Yes, I will remove it.

Reply:
Actually, reverted the new commit. So it is needed now.